src/screens/index.tsx

```diff
@@ -375,12 +375,14 @@ function Blocks() {
       >
         {virtualizer.getVirtualItems().map(({ key, index, size, start }) => {
           const { block, status } = blocks[index] || {}
+          const isPending = status === 'pending'
           if (!block) return
           return (
             <Link
               onClick={() => setPosition(parentRef.current?.scrollTop!)}
               to={`block/${block.number}`}
               key={key}
+              style={{ pointerEvents: isPending ? 'none' : undefined }}
```

Comment:
We might need the `pointerEvents` attribute in future things! Could be nice to add it to the `Box` properties [here](https://github.com/paradigmxyz/rivet/blob/main/src/design-system/components/Box.css.ts#L73).